.github/workflows/report-stable.yml

```diff
@@ -0,0 +1,24 @@
+
+name: Report-API-Updates
+
+# Run tests on pull requests and when changes are directly
+# commited to master.
+on:
+  workflow_dispatch: {}
```

Comment:
So this means a manual trigger?

Reply:
Yep. I didn't want this one to run automatically. :-)